Rego/AADConfig.rego

```diff
@@ -15,6 +15,10 @@ Description(String1, String2, String3) =  trim(concat(" ", [String1, String2, St
 
 ReportDetailsArray(Array, String) = Description(Format(Array), String, "")
 
+ReportDetailsBoolean(Status) = "Requirement met" if {Status == true}
+
+ReportDetailsBoolean(Status) = "Requirement not met" if {Status == false}
+
```

Comment:
Use ReportDetailsBoolean from the Rego/Utils/ReportUtils.rego file instead of duplicating here.